index.html

```diff
@@ -39,17 +39,142 @@ <h3 class="connect">LET'S CONNECT</h3>
+        </div>
+      </div>
+
+      <div class="projects">
+        <img src="./images/Snapshoot Portfolio(3).png" alt="">
+        <h2 class="works-title">TONIC</h2>
+        <p><span class="canopy">Canopy</span> <img src="./images/Counter.png" alt="Counter"> <span class="dev"> End Dev
+            <img src="./images/Counter.png" alt="Counter"> 2015</span></p>
+        <div class="project-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div>
+          <button class="languages">HTML</button>
+          <button class="languages">CSS</button>
+          <button class="languages">JAVASCRIPT</button>
+        </div>
+        <div>
+          <button class="see-project">See Project</Button>
+        </div>
+      </div>
+
+      <section class="about">
+        <div>
+          <div class="first-heading">
+            <h1>About Myself</h1>
+          </div>
+          <div class="description">
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+              work
+              and
+              experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+          </div>
+          <div>
+          <h3 class="connect">LET'S CONNECT</h3>
+          <ul class="social">
+            <li class="social"><a href="#"><img src="./images/Vector.svg" alt=""></a></li>
+            <li class="social"><a href="#"><img src="./images/Linkedin icon.svg" alt="Linkedin"></a></li>
+            <li class="social"><a href="#"><img src="./images/Vector (1).svg" alt=""></a></li>
+            <li class="social"><a href="#"><img src="./images/Vector (2).svg" alt="github"></a></li>
+            <li class="social"><a href="#"><img src="./images/Vector (3).svg" alt="twitter"></a></li>
+          </ul>
+        </div>
+          <div>
+            <button class="see-project">Get my resume</Button>
+          </div>
+          <div class="union-div">
+            <h2>Languages</h2>
+            <img src="./images/Union.png" alt="">
+          </div>
+          <div class="about-lang">
+            <img src="./images/Ellipse 1.png" alt="">
+            <a href="http://#">JavaScript</a>
+          </div>
+          <div class="about-lang">
+            <img src="./images/Ellipse 1 (1).png" alt="">
+            <a href="http://#">HTML</a>
+          </div>
+          <div class="about-lang">
+            <img src="./images/Ellipse 1 (2).png" alt="">
+            <a href="http://#">CSS</a>
+          </div>
+          <div class="union-div">
+            <h2>Frameworks</h2>
+            <img class="union2" src="./images/Union2.png" alt="">
+          </div>
+          <div class="union-div">
+            <h2>Skills</h2>
+            <img class="union2 space" src="./images/Union2.png" alt="">
+          </div>
+        </div>

```

Comment:
- [ Required ] I think it would be better to turn this list of technologies to be a `ul` which contains nested `ul` tags for it to be semantically correct and accessible to users using web accessibility tools like screen readers. Please see the example code for an example of this, this is just a suggestion if you have another idea please feel free to implement it.

```HTML
          <ul class="union-div">
            <li> 
            <h2>Languages</h2>
            <img src="./images/Union.png" alt="">
          <li class="about-lang">
            <img src="./images/Ellipse 1.png" alt="">
            <a href="http://#">JavaScript</a>
          </li>
          <li class="about-lang">
            <img src="./images/Ellipse 1 (1).png" alt="">
            <a href="http://#">HTML</a>
          </li>
          <li class="about-lang">
            <img src="./images/Ellipse 1 (2).png" alt="">
            <a href="http://#">CSS</a>
            </li>
          </ul>
           </li>
          <li class="union-div">
            <h2>Frameworks</h2>
            <img class="union2" src="./images/Union2.png" alt="">
          </li>
          <li class="union-div">
            <h2>Skills</h2>
            <img class="union2 space" src="./images/Union2.png" alt="">
          </li>
        </ul>
```